apis/crd/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -23,6 +23,36 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *AwsAccountCredential) DeepCopyInto(out *AwsAccountCredential) {
+	*out = *in
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new AwsAccountCredential.
+func (in *AwsAccountCredential) DeepCopy() *AwsAccountCredential {
```

Comment:
I don't quite get it? You mean you want to have type def in some separate file in the future? I placed them here because webhook is going to need them